vaadin-icons-flow-parent/vaadin-icons-flow/src/main/java/com/vaadin/flow/component/icon/Icon.java

```diff
@@ -77,14 +80,8 @@ public Icon(String icon) {
      *            the icon collection
      * @param icon
      *            the icon name
-     * @deprecated Use either {@link #Icon(String)} or
```

Comment:
Removed the deprecation message from here which suggests to use IronIcon component because IronIcon will be deprecated in https://github.com/vaadin/flow-components/issues/884

Reply:
The reason why this constructor was deprecated in https://github.com/vaadin/vaadin-icons-flow/pull/57 was that if one is providing a collection name, they should use an alternative to `Icon` (`IronIcon`) which is identical for all purposes, except for it does not automatically load `vaadin-iconset` and requires the developers to load the icon set themselves.

When using this constructor, the developers need to load the `collection` icon set themselves. Should the docs mention that? What do you think?

Reply:
Added documentation for it here https://github.com/vaadin/flow-components/pull/1785/commits/d703bdcc8a5d7f8bf7132a68b470f2be19214eac